### ISSUE TITLE
CI: bundletester 40-security-check failure in 1.10

### ISSUE THREAD

**Issue by kwmonroe** [juju-solutions/bundle-canonical-kubernetes#531]
K8s-core with 1.10/edge failed `test_bundletester`:

https://ci.kubernetes.juju.solutions/job/test-cdk/466/testReport/junit/(root)/bundle/40_security_check_py/

```
======================================================================
FAIL: test_basic (__main__.IntegrationTest)
Test the effect of changing the admin password
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/bundletester-LT1C8I/canonical-kubernetes/tests/40-security-check.py", line 77, in test_basic
    self.assertEqual(status, 200)
AssertionError: 404 != 200
```

Charms were from the beta channel:
```
App                Version  Status  Scale  Charm              Store       Rev  OS      Notes
easyrsa            3.0.1    active      1  easyrsa            jujucharms   38  ubuntu
etcd               3.2.9    active      1  etcd               jujucharms   76  ubuntu
flannel            0.9.1    active      2  flannel            jujucharms   51  ubuntu
kubernetes-master  1.10.0   active      1  kubernetes-master  jujucharms   99  ubuntu  exposed
kubernetes-worker  1.10.0   active      1  kubernetes-worker  jujucharms  111  ubuntu  exposed
```

Snaps on k8s-master were from edge:
```
$ snap list
Name                     Version    Rev   Developer  Notes
cdk-addons               1.10.0     378   canonical  -
core                     16-2.31.2  4206  canonical  core
kube-apiserver           1.10.0     385   canonical  -
kube-controller-manager  1.10.0     375   canonical  -
kube-scheduler           1.10.0     384   canonical  -
kubectl                  1.10.0     384   canonical  classic
```

**Comment by kwmonroe** [comment]
FYI, 40-security-check.py lives in the bundletester tests:

https://github.com/juju-solutions/bundle-canonical-kubernetes/blob/master/tests/40-security-check.py

**Comment by kwmonroe** [comment]
Closed with #532
